### PR TITLE
Create package

### PR DESCRIPTION
**Pull request by stackptr**
`@freckle/maybe` comprises helpers that are used to operate on nullable types. The operations on this type are influenced heavily by the [`Data.Maybe` Haskell package][0], with some affordances for JavaScript quirks.

The `README.md` provides documentation for using these operations. 

[0]: https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Maybe.html

### CODE REVIEW

File: .restyled.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+restylers:
+  - clang-format:
+	  enabled: false
```
Comment (pbrisbin) [inline .restyled.yaml:3]: I think this red highlighting on GitHub is indicating actual-tabs?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2021 Renaissance Learning Inc
```
Comment (pbrisbin) [inline LICENSE:3]: Are we 2022 these days!?

**Review by pbrisbin — APPROVED** [review]
Yup, that's JavaScript alright.

File: .restyled.yaml (hunk 1 above)
Comment (stackptr) [reply .restyled.yaml]: Not sure how that got in there.

**Review by joris974 — APPROVED** [review]
I wonder if we should make `lodash` a peer dependency. That is maybe another battle.

**Comment by stackptr** [comment]
> I wonder if we should make `lodash` a peer dependency. That is maybe another battle.

I don't think this is necessary, different versions of `lodash` among dependencies should be fine: https://nodejs.org/es/blog/npm/peer-dependencies/

**Pull request merged**